spec/collator.html

```diff
@@ -35,6 +35,10 @@ <h1>InitializeCollator ( _collator_, _locales_, _options_ )</h1>
         1. Set _opt_.[[kn]] to _numeric_.
```

Comment:
Is it necessary to add the following line from #175?

```



      <p>
        The following algorithm refers to the `type` nonterminal from <a href="http://www.unicode.org/reports/tr35/#Unicode_locale_identifier">UTS 35's Unicode Locale Identifier grammar</a>.
      </p>
```

Reply:
done